apps/condo/domains/organization/integrations/sbbol/sync/syncBankAccounts.js

```diff
@@ -0,0 +1,81 @@
+const { getLogger } = require('@open-condo/keystone/logging')
+const { get, isEmpty } = require('lodash')
+
+const { initSbbolFintechApi } = require('@condo/domains/organization/integrations/sbbol/SbbolFintechApi')
+const { dvSenderFields } = require('@condo/domains/organization/integrations/sbbol/constants')
+const { BankAccount } = require('@condo/domains/banking/utils/serverSchema')
+const { RUSSIA_COUNTRY } = require('@condo/domains/common/constants/countries')
+const { ISO_CODES_FOR_SBBOL } = require('@condo/domains/common/constants/currencies')
+const { getSchemaCtx } = require('@open-condo/keystone/schema')
+const { Organization } = require('@condo/domains/organization/utils/serverSchema')
+
+const logger = getLogger('sbbol/syncBankAccounts')
+
+const _syncBankAccount = async (ClientInfo) => {
+    const { keystone: userContext } = await getSchemaCtx('User')
+    const { keystone: bankAccountContext } = await getSchemaCtx('BankAccount')
+    const [organization] = await Organization.getAll(userContext,
+        { tin: ClientInfo.inn },
+        { sortBy: 'createdAt_DESC', first: 1 },
+    )
+
+    for (const account of ClientInfo.accounts) {
+        const bankAccountDetails = {
+            tin: organization.tin,
+            country: RUSSIA_COUNTRY,
+            number: account.number,
+            currencyCode: (ISO_CODES_FOR_SBBOL[account.currencyCode]),
+            routingNumber: account.bic,
+        }
+        const foundAccount = await BankAccount.getOne(bankAccountContext, {
+            ...bankAccountDetails,
+            organization: { id_in: organization.id },
+        })
+
+        if (!foundAccount) {
+            logger.info('BankAccount instance created')
```

Comment:
this info is not very helpful in logs, maybe it would be better if we add some more info, like organization tin, account type, etc.